### ISSUE TITLE
Able to view boat ramps marked as private?

### ISSUE THREAD

**Issue by swaggermeister** [ChromeTemplar/nhlakes_unhm#29]
I would think only the owner/creator should be able to see ramps marked private? Or an admin user, which doesn't exist yet.

**Comment by AvenZrk** [comment]
I don't think that is meant as private viewing of the data, but ownership of land that the boat ramp resides:
its on public or private land > from what I understand

**Comment by ChromeTemplar** [comment]
Did you check with Andrea on this?
